src/libs/migrations/ReformatTimezone.js

```diff
@@ -0,0 +1,41 @@
+import Onyx from 'react-native-onyx';
+import _ from 'underscore';
+import ONYXKEYS from '../../ONYXKEYS';
+import CONST from '../../CONST';
+
+// This migration changes the format of the timezone in the Onyx key MY_PERSONAL_DETAILS from a string to an object
+export default function () {
+    return new Promise((resolve) => {
+        // Connect to the old key in Onyx to get the old value of myPersonalDetails timezone
+        // then update the timezone to be the default timezone and set the myPersonalDetails
+        // key with the updated values
+        const connectionID = Onyx.connect({
+            key: 'myPersonalDetails',
```

Comment:
Why can't we use `ONYXKEYS.MY_PERSONAL_DETAILS` here?
```suggestion
            key: ONYXKEYS.MY_PERSONAL_DETAILS,
```

Reply:
nice catch!